build.gradle

```diff
@@ -138,6 +138,10 @@ dependencies {
     testImplementation "org.mockito:mockito-core:3.7.7"
 
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitVersion"
+
+    testCompile "org.mockito:mockito-junit-jupiter:3.8.0"
+    testCompile "org.mockito:mockito-core:3.8.0"
```

Comment:
There is a dependency to mockito-core defined 5 lines above. 
Also `testCompile` seems to be deprecated in recent Gradle versions so I believe it should be `testImplementation`.
And could you please move the version to a variable like it's done for other libraries?